### PR TITLE
Updating ose-cluster-authentication-operator images to be consistent with ART

### PR DESCRIPTION
**Pull request by s-urbaniak**
supersedes #543

### CODE REVIEW

File: _output/named-golang-versions [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.ci-operator.yaml: 1.18
```
Comment (stlaz) [inline _output/named-golang-versions:1]: eh, remove the `_output`

**Review by stlaz — CHANGES_REQUESTED** [review]
fix https://github.com/openshift/cluster-authentication-operator/blob/master/.ci-operator.yaml

edit: I see it's fixed, I accidentally submitted my review too early

File: _output/named-golang-versions (hunk 1 above)
Comment (s-urbaniak) [reply _output/named-golang-versions:1]: lol, good catch

Comment (s-urbaniak) [reply _output/named-golang-versions]: done

**Pull request merged**